doku/src/printers/toml/output/layouts.rs

```diff
@@ -0,0 +1,4 @@
+pub mod one_column;
+pub mod two_columns;
+
+use super::*;
```

Comment:
It looks like both layouts are identical for JSON & TOML, right? If so, maybe we could move them to some `mod common;`?